packages/components/src/DateTimePickers/DateTime/date-extraction.js

```diff
@@ -423,6 +423,21 @@ function extractPartsFromDateTime(datetime, options) {
 	};
 }
 
+/**
+ * extract date/time part from datetimeformat
+ * @param {string} datetimeFormat
+ * @param {string} type
+ */
+function extractPartsFromDateTimeFormat(datetimeFormat, type) {
+	const formats = datetimeFormat.split(' ');
+	if (type === 'date') {
+		return formats[0];
+	} else if (formats.length === 2) {
```

Comment:
This doesn't work if user add multiple blanks.
I think we can do a little better: 
```javascript
const formatParts = dateTimeFormat
    .split(/\s/g) // split from blank
    .filter(part => part); // remove empty elements

if (type === 'date') {
     return formats[0] || ''; 
}
return formats[1] || '';
```
Then you can return the first or second element.

Reply:
changed as suggested.